packages/cmake.rb

```diff
@@ -3,23 +3,24 @@
 class Cmake < CMake
   description 'CMake is an open-source, cross-platform family of tools designed to build, test and package software.'
   homepage 'https://cmake.org/'
-  version '3.29.0'
+  version '3.29.2'
   license 'CMake'
   compatibility 'all'
   source_url 'https://github.com/Kitware/CMake.git'
   git_hashtag "v#{version}"
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: 'dbbbb44e2651a607c690b545b91c3749fd5395fb7a53657c6d0703f6133d55b1',
-     armv7l: 'dbbbb44e2651a607c690b545b91c3749fd5395fb7a53657c6d0703f6133d55b1',
-       i686: '1473e3e8f16ea34a841ec585ad31cf3987d0a38e1cbfe5c250ef689667f30df5',
-     x86_64: '723369fb9a2d4758d61ac98a7aad87c7ae0e3028df531d789a5284a12973d060'
+    aarch64: '06d2f226d1eff483d29ec8d861fb2c47a0200f82d30f121d872f89975ff70c2f',
+     armv7l: '06d2f226d1eff483d29ec8d861fb2c47a0200f82d30f121d872f89975ff70c2f',
+       i686: 'c4715d535d0637d37f4c5c7d7ae4a9f903baea88d8221848f9cc8ae9625781af',
+     x86_64: '5ad0c9ee32f7c058a6e31954b74d47cfa339f3c1184f00e68a9fa1100f1254af'
   })
 
   depends_on 'bzip2' => :build
   depends_on 'cppdap' # R
   depends_on 'curl' # R
+  depends_on 'distcc' => :build
```

Comment:
Does this really need to be the case?

Reply:
Just trying to speed up the build. Tests take forever... Using distcc helps...